guides/common/modules/proc_migrating-a-host-to-use-pull-mode.adoc

```diff
@@ -0,0 +1,34 @@
+[id="migrating-a-host-to-use-pull-mode_{context}"]
```

Comment:
I think this is sufficiently covered by https://github.com/theforeman/foreman-documentation/pull/1548 so this file can be removed.